### PR TITLE
adds a logger tracer

### PR DESCRIPTION
**Pull request by jcarres-mdsol**
It is always difficult to know what we would be sending.
This tracer let us see what would be sent. It also may be used if someone wants to send zipkin info to the logs instead of directly to the server.
@jfeltesse-mdsol  @ykitamura-mdsol

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,14 @@ The original tracer, it uses scribe and thrift to send traces.
 
 You need to explicitly install the gem (`gem 'scribe', '~> 0.2.4'`) and set `:scribe_server` in the config.
 
+### Logger
+
+The simplest tracer that does something. It will log all your spans using the logger you pass in the configuration.
+This tracer can be used for debugging purpose (to see what is going to
+be sent) or to deliver zipkin information into the logs for later
+retrieval and analysis.
```
Comment (jfeltesse-mdsol) [inline README.md]: [nits] line breaking is kinda strange

File: README.md [hunk 2]
```diff
@@ -105,6 +105,14 @@ The original tracer, it uses scribe and thrift to send traces.
...
+
+You need to set `:logger` in the configuration.
```
Comment (jfeltesse-mdsol) [inline README.md]: to what value?

File: lib/zipkin-tracer/config.rb [hunk 3]
```diff
@@ -29,6 +30,8 @@ def adapter
         :scribe
       elsif present?(@zookeeper) && RUBY_PLATFORM == 'java'
         :kafka
+      elsif @logger_setup
```
Comment (jfeltesse-mdsol) [inline lib/zipkin-tracer/config.rb]: what happens if the user sets `logger: ""` ?

**Comment by jfeltesse-mdsol** [comment]
I'm not sure how to use this, as a user what do I need to set in my config file?

File: README.md (hunk 2 above)
Comment (jcarres-mdsol) [inline README.md]: it is already explained in the README, the :logger key needs to get an object

File: lib/zipkin-tracer/config.rb (hunk 3 above)
Comment (jcarres-mdsol) [inline lib/zipkin-tracer/config.rb]: that's a bug. We say logger needs an logger object

**Comment by jcarres-mdsol** [comment]
The README already especifies how to use logger, we are just saying that it needs to be set

File: lib/zipkin-tracer/config.rb (hunk 3 above)
Comment (jfeltesse-mdsol) [inline lib/zipkin-tracer/config.rb]: :ok_woman:

**Comment by jfeltesse-mdsol** [comment]
need to amend the commit message: "**t** adds a logger tracer"... :sweat_smile:

File: Rakefile [hunk 4]
```diff
@@ -116,6 +116,7 @@ task :benchmark do
   null_configuration = { sample_rate: 1 }
   json_configuration = null_configuration.merge({ json_api_host: fake_url })
   scribe_configuration = null_configuration.merge({ scribe_server: fake_url })
+  logger_configuration = null_configuration.merge({ logger: logger })
```
Comment (jfeltesse-mdsol) [inline Rakefile]: [nits] you could do without the braces in there

**Pull request merged**